src/toml.rs

```diff
@@ -473,6 +473,14 @@ impl TomlParser {
                     while self.cur != '\n' && self.cur != '\0' {
                         self.next(i);
                     }
+
+                    while self.cur == '\n'
```

Comment:
I feel like those two while loops could be merged together?

Reply:
No we can't cause it fails in the following test. After the second `#` we have an empty line.
```
[[array]]
# a
name = "a" #aasdf

[[array]]
```

Reply:
The first loop removes any characters except LF.
The second loop finds the first char which is not space.

So I don't think we can merge them.